lightning/src/ln/channelmanager.rs

```diff
@@ -3138,6 +3165,93 @@ impl<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelManager<M, T, K, F
 		Ok(())
 	}
 
+	/// Attempts to forward an intercepted HTLC over the provided channel id and with the provided
+	/// amount to forward. Should only be called in response to an [`HTLCIntercepted`] event.
+	///
+	/// Intercepted HTLCs can be useful for Lightning Service Providers (LSPs) to open a just-in-time
```

Comment:
Grepping this is the first time we mention LSP in the codebase. I think it could be opportunistic to make an entry in `GLOSSARY.md`, at least to document how "we" LDK contributors understand it. The reality behind varies a lot across the ecosystem and it could be good to have a well-defined term to make sense of API like this one.

Reply:
Not opposed but sounds a bit bikeshed-y, would prefer to punt for now